packages/edit-site/src/components/global-styles/style.scss

```diff
@@ -243,6 +248,10 @@
 	color: inherit;
 }
 
+.edit-site-global-styles-screen-root.edit-site-global-styles-screen-root {
+	box-shadow: none;
+}
```

Comment:
I noticed that there is a prop called [`isBorderless`](https://github.com/WordPress/gutenberg/blob/5342773f00f6a83a4995c9e98a30252c3b93677e/packages/components/src/card/card-header/README.md#isborderless-boolean) to not display a border on the `Card` component.

Can you remove this style and add the `isBorerless` prop [here](https://github.com/WordPress/gutenberg/blob/5342773f00f6a83a4995c9e98a30252c3b93677e/packages/edit-site/src/components/global-styles/screen-root.js#L60)?

This approach is also used in the variations panel:

https://github.com/WordPress/gutenberg/blob/5342773f00f6a83a4995c9e98a30252c3b93677e/packages/edit-site/src/components/global-styles/screen-style-variations.js#L45-L49